package.json

```diff
@@ -38,8 +38,8 @@
     "mkdirp": "^0.5.1",
     "pify": "^3.0.0",
     "tmp": "0.0.29",
-    "uglify-es": "^3.0.26",
-    "uglify-js": "^3.0.21",
+    "uglify-es": "^3.3.9",
+    "uglify-js": ">3.4 <4",
```

Comment:
I think `^3.5.0` will have the same effect. You can try it out here: https://semver.npmjs.com/